quantum/keycode_config.h

```diff
@@ -44,6 +44,7 @@ typedef union {
         bool oneshot_enable : 1;
         bool swap_escape_capslock : 1;
         bool autocorrect_enable : 1;
+        uint8_t autocorrect_curr_dict : 3;
```

Comment:
This (and respective check on the Py code-gen) should perhaps be lowered to 2 or even 1 bit, to not exhaust all available space in current EEPROM footprint.